Exercise_merge_conflict/Exercise_6.ipynb

```diff
@@ -0,0 +1,512 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20afb074",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Go back to main branch and try to merge the updated_schedules branch\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "58fe9568",
+   "metadata": {},
+   "source": [
+    "Now try to solve the merge conflict with ```git restore schedule_day1 --theirs/ours```."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cb2c271a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Restore the version of your favorite branch\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e1a4bcd9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Commit the merge with your favorite solution (don't forget to add the solution first)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "20bc5bca",
+   "metadata": {},
+   "source": [
+    "### Merge conflict when trying to pull remote branch\n",
+    "We just realized that we forgot an introduction talk to the conference. Switch to the updated_schedules branch and change the first event to \"Introduction\" in schedule_day1 and commit your changes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "81486f4d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Switch to updated_schedules\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aef5c7ae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Change the first event to \"Introduction\" and commit your changes\"\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c5b0d1f6",
+   "metadata": {},
+   "source": [
+    "What we didn't notice is that someone changed something on the remote branch while we were working on the file. When we now pull try to pull the remote, we will again run into a merge conflict."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "66f617f5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This command will do that \"unnoticed\" change on the remote branch\n",
+    "commit_to_remote"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0c247138",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make sure you are on the updated_schedules branch and try to pull with \"git pull\"\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1eb1c0a",
+   "metadata": {},
+   "source": [
+    "Unless you already set how to pull in your git config, you will get a message like this:\n",
+    "```\n",
+    "hint: You have divergent branches and need to specify how to reconcile them.\n",
+    "hint: You can do so by running one of the following commands sometime before\n",
+    "hint: your next pull:\n",
+    "hint: \n",
+    "hint:   git config pull.rebase false  # merge\n",
```

Comment:
We need to explain the other option to the users as well.

Or we just set ```git config pull.rebase false```silently in the background.

I did not know about the other options too

Reply:
I added a description of all options.